riscv-c-api.md

```diff
@@ -203,6 +203,100 @@ function declares more than one mode or an undefined mode.
 
 This attribute is incompatible with the `naked` attribute.
 
+### `__attribute__((target("<ATTR-STRING>")))`
+
+The `target` attribute is used to enable a set of features or extensions for a
+function.
+
+For instance, you can enable the `v` extension for a specific function even if
+the `-march` or `-mcpu` options do not include the `v` extension. Importantly,
+this won't alter the global settings. Here is an example:
+
+```c
+__attribute__((target("arch=+v")))
+int foo(int a)
+{
+  return a + 5;
+}
+```
+
+Using the `target` attribute for a function should not affect the file scope
```

Comment:
"file scope" should maybe be "translation unit scope" since files can include other files. The global attributes apply to every file being included.